paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -695,10 +695,16 @@ def valid_transitions(self) -> Iterator[state_machine.TransitionDefinition]:
     def disable_auto_rollbacks(self):
         self.cancel_auto_rollback_countdown()
         self.auto_rollback = False
+        self.update_slack_status(
+            f"Automatic rollback disabled for this deploy. To disable this permanently for this step, `edit deploy.yaml`"
+        )
 
     def enable_auto_rollbacks(self):
         self.auto_rollback = True
         self.auto_rollbacks_ever_enabled = True
+        self.update_slack_status(
+            f"Automatic rollback enabled for this deploy. Will watch for failures and rollback when necessary"
```

Comment:
Maybe also mention setting `auto_rollbacks` in `deploy.yaml` in this message too.